cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -80,32 +88,35 @@
     private static final String LOG_TAG = "BasicDatastore";
     private static final Logger logger = Logger.getLogger(DatastoreImpl.class.getCanonicalName());
 
-    private static final String FULL_DOCUMENT_COLS = "docs.docid, docs.doc_id, revid, sequence, json, current, deleted, parent";
+    public static final String FULL_DOCUMENT_COLS = "docs.docid, docs.doc_id, revid, sequence, " +
+            "json, current, deleted, parent";
 
-    private static final String GET_DOC_NUMERIC_ID =
+    public static final String GET_DOC_NUMERIC_ID =
```

Comment:
I think placing it in the callable will make it easier to understand whats going on, and it has the bonus of making people use the callable rather than just run the SQL from the constants class.